Library/Formula/agda.rb

```diff
@@ -9,6 +9,7 @@ class Agda < Formula
   mirror "https://github.com/agda/agda/archive/2.4.2.3.tar.gz"
   sha256 "bc6def45e32498f51863d67acfbe048c039d630c6a36761ed27e99a5f68d7b27"
   revision 2
+  head "https://github.com/agda/agda.git", :branch => "master"
```

Comment:
`master` never needs specifying on `git` branches. We assume that's the default & don't need pointing unless it isn't.